.github/workflows/mops-publish.yml

```diff
@@ -16,4 +17,3 @@ jobs:
           identity-pem: ${{ secrets.MOPS_IDENTITY_PEM }}
       - run: mops install
       - run: mops publish
-      
```

Comment:
can we re-add this NL?

Reply:
Yep; the newline has no effect (this was originally a community contribution which removed newlines from various other files).